wiki/Client/Beatmap_editor/Menu/en.md

```diff
@@ -156,18 +148,18 @@ Places of interest.
 
 | Name | Description |
 | :-- | :-- |
-| Snap Divisor | [Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) |
-| Audio Rate | Same as Playback Rate. |
-| Grid Snapping (T) | Allow snapping of the notes with respect to current grid line. |
+| Snap Divisor | Change the granularity of the [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor), displaying more or fewer ticks on the [timeline](/wiki/Client/Beatmap_editor/Timelines). |
+| Audio Rate | Change the playback rate of the audio. |
+| Grid Snapping (T) | Toggle [snapping](/wiki/Beatmapping/Grid_snapping) of the hit objects to the playfield grid. |
 
 ### Special commands
 
 | Name | Description |
 | :-- | :-- |
-| Create Polygon Circles (`Ctrl` + `Shift` + `D`) | Create a circular flow of hit circles. You can adjust the Distance Snap (0.1x - 2.0x) {Spacing between notes}, Offset angle (0° - 180°) {Rotation angle}, Repeat and Points (3 - 14/282) {Number of hit circle}. |
-| Convert slider to stream... | (Select a slider) Change the slider to streams of hit circles. You can select between two options: By object count or By distance snap. **By object count:** Adjust the slider flow with the given amount of hit circles (1 - 17/100). **By distance snap:** Adjust the slider flow with a set distance between two notes (0.10x - 5.00x). Can result in absurd amount of hit circles, since it relies on Beat Snap Divisor (one beat = one note). **Beat Snap Divisor:** Note timing adjuster. |
-| Enable live mapping mode (`Ctrl` + Tab) | osu!/osu!catch/osu!taiko maps: use osu!taiko keys to input hit circles, fruits, or taiko hit objects, while playing the song in editor. Placed via cursor location. Different hitsounds based on what osu!taiko key you pressed. osu!mania maps: use osu!mania keys to input hit objects, while playing song in editor. Placed via the key you pressed. Does not work for sliders/holds. |
-| Sample import | [Keysound and Sound Sampling.](/wiki/Client/Beatmap_editor/Compose) |
+| Create Polygon Circles... (`Ctrl` + `Shift` + `D`) | Create a regular polygon consisting of several hit circles, using active beat snap divisor for circle placement. |
+| Convert slider to stream... | Replace the [body](/wiki/Gameplay/Hit_object/Slider/Sliderbody) of a selected slider with a [stream](/wiki/Beatmap/Pattern/osu!/Stream) of hit circles. **By object count:** use the exact amount of hit circles. **By distance snap:** use the exact [distance snap](/wiki/Client/Beatmap_editor/Distance_snap) between hit circles. |
+| Enable live mapping mode (`Ctrl` + `Tab`) | osu!/osu!catch/osu!taiko maps: use osu!taiko keys to place hit circles or fruits while playing the song in editor. The circles are placed under the cursor, and different hitsounds are applied based on the key pressed. osu!mania maps: use osu!mania keys to place notes while playing song in editor. Notes are placed in the column corresponding to the key pressed. Does not work for sliders/holds. |
```

Comment:
```suggestion
| Enable live mapping mode (`Ctrl` + `Tab`) | Place hit objects with gameplay keybinds while the song is playing. Does not work for osu! sliders and osu!mania hold notes. |
```

Reply:
I think this deserves a footnote because the original explanation, although wordy, is still valuable